src/v0/destinations/intercom/transform.js

```diff
@@ -181,7 +188,7 @@ function validateAndBuildResponse(message, payload, category, destination) {
     case EventType.GROUP: {
       response.body.JSON = removeUndefinedAndNullValues(buildCustomAttributes(message, payload));
       respList.push(response);
-      if (checkIfEmailOrUserIdPresent(message)) {
+      if (checkIfEmailOrUserIdPresent(message, destination.Config)) {
         const attachUserAndCompanyResponse = attachUserAndCompany(message, destination.Config);
         attachUserAndCompanyResponse.userId = message.anonymousId;
```

Comment:
Will this always be mapped to anonymousId ?

Reply:
This will not be sent to Intercom